### ISSUE TITLE
Minitiwt accept twits from users not created

### ISSUE THREAD

**Issue by Thomsen96** [DevelOpsITU/MiniTwit#85]
From the return code it looks like it through the /sim endpoint is possible to publish messages from useres that doesnot exits

Image is from just after #83 and is started with a completly clean database.
![billede](https://user-images.githubusercontent.com/32035863/156420498-70692dcc-eff8-49bc-93b9-d253c0796eed.png)

**Comment by Lucki2g** [comment]
How do you know the user does not exists? As far as I see in the code, we do check if the user exists before trying to add the message. In case of err we return badrequest.
![image](https://user-images.githubusercontent.com/11597762/156928996-b0fc9f37-a54a-43cf-aef1-539e07ab6f61.png)
![image](https://user-images.githubusercontent.com/11597762/156929002-c99ad8f8-8152-4766-bc53-d9b35f7a14fd.png)

**Comment by Thomsen96** [comment]
In this instance I know it. Because it was just after #83. Where the docker container had crashed and with it did we loose the database file. 

To get the application online again and not loose the database did the following happen:
- Move database outside of the docker container 
- Upload an empty database to the Application server
- Start the minitiwt application with the fresh database

But nice that it seems like we have fixed it because on the current version of minitwit, running on the server are we not verifying the user existence.: https://github.com/DevelOpsITU/MiniTwit/blob/237df007612e4e70bde2fe3f4401f34f75f12d52/database/sqlite3.go#L247-L267

Current version of minitwit that we have published 
![billede](https://user-images.githubusercontent.com/32035863/156930608-945f8fa8-bb98-4747-8757-f0c236ba3eb0.png)
